+precice/@SolverInterface/private/preciceGateway.cpp

```diff
@@ -313,6 +316,34 @@ class MexFunction: public matlab::mex::Function {
                 interface->setMeshQuadWithEdges(meshID[0],firstVertexID[0],secondVertexID[0],thirdVertexID[0],fourthVertexID[0]);
                 break;
             }
+            case FunctionID::setMeshAccessRegion:
+            {
+                const TypedArray<int32_t> meshID = inputs[1];
+                const TypedArray<double> boundingBox = inputs[2];
+                interface->setMeshAccessRegion(meshID[0],&*boundingBox.begin());
+                break;
+            }
+            case FunctionID::getMeshVerticesAndIDs:
+            {
+                const TypedArray<int32_t> meshID = inputs[1];
+                const TypedArray<int32_t> size = inputs[2];
+                buffer_ptr_t<int32_t> ids_ptr = factory.createBuffer<int32_t>(size[0]);
+                int32_t* ids = ids_ptr.get();
+                buffer_ptr_t<double> positions_ptr = factory.createBuffer<double>(size[0]);
+                double* positions = positions_ptr.get();
+                interface->getMeshVertices(meshID[0],size[0],ids,positions);
+                outputs[0] = factory.createArrayFromBuffer<double>({1,size[0]}, std::move(positions_ptr));
+                outputs[1] = factory.createArrayFromBuffer<int32_t>({1,size[0]}, std::move(ids));
+                break;
+            } 
+            
```

Comment:
```suggestion
```
Again: Consistency, but I think there are also a few places where this is different, below. You can also ignore this. As long as we don't have a stylechecker it does not make a lot of sense investing effort here (if it is just about blank lines and spaces).